.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,60 @@
+name: Bug report
```

Comment:
Do you want a `refactor.yml` issue form? It would be copy/pasted/adjusted from the one we have over on the main Renovate repository.